tests/test_pdf_cb.py

```diff
@@ -91,7 +91,7 @@ def test_cb_dcb():
     assert not any(np.isnan(probsr))
 
     probsl_scipy = crystalball.pdf(x, beta=alphal, m=nl, loc=mu, scale=sigma)
-    probsr_scipy = crystalball.pdf(-x, beta=alphar, m=nr, loc=mu, scale=sigma)
+    probsr_scipy = crystalball.pdf(-x + 2 * mu, beta=alphar, m=nr, loc=mu, scale=sigma)
```

Comment:
Is this dark magic :D ?

Reply:
Haha kinda: we want to mirror it at the mu. x is not at mu, we mirror it (now at -mu), shift it back.

Simple math thing, mirroring at point mu

Reply:
Oh yeah cause mu is not equal to zero anymore.

Reply:
yep, just breaking some symmetries to see things still work